client/run.go

```diff
@@ -290,6 +303,67 @@ func (cli *HyperClient) JsonFromCmdline(cmdArgs, cmdEnvs, cmdPortmaps []string,
 	return string(jsonString), nil
 }
 
+func parseVolume(volStr string) (*pod.UserVolume, *pod.UserVolumeReference, error) {
```

Comment:
make sense, do you need me also update the parsePortMapping routine? if no need. please help close the PR, thanks.